spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -232,6 +232,8 @@ public MessageProperties toMessageProperties(AMQP.BasicProperties source, Envelo
 
 	private volatile int defaultTxSize = DEFAULT_TX_SIZE;
 
+	protected volatile boolean defaultDurableSubscription = true;
+
```

Comment:
Actually, there is another reason for doing it this way. If we make this non-durable, tap subscribers in SCDF will not be able to create durable subscriptions easily (i.e. without passing rabbit-specific properties), since there is no global property that governs that. Given that an SCS user can opt for either, I think that the defaults should cover the most likely to be used use cases.